boards/shields/rpi_pico_lcd/doc/waveshare_pico_lcd_1_8/pinouts.rsti

```diff
@@ -0,0 +1,89 @@
+       - | :rpi-pico-pin:`11` PIO/PWM : :rpi-pico-pio:`GP8` (PWM8)
+         | :rpi-pico-pin-nc:`nc` on-shield LCD data/cmd :rpi-pico-sys:`LCD_DC`
+       - | :rpi-pico-pin:`12` :rpi-pico-spi:`SPI1_CSN` : :rpi-pico-pio:`GP9` (PWM9)
+         | :rpi-pico-pin-nc:`nc` on-shield LCD chip select :rpi-pico-sys:`LCD_CS`
+       - | :rpi-pico-pin:`13` :rpi-pico-gnd:`GND`
+       - | :rpi-pico-pin:`14` :rpi-pico-spi:`SPI1_SCK` : GP10 (PWM10)
+         | :rpi-pico-pin-nc:`nc` on-shield LCD i/f clock :rpi-pico-sys:`LCD_CLK`
+       - | :rpi-pico-pin:`15` :rpi-pico-spi:`SPI1_TX` : GP11 (PWM11)
+         | :rpi-pico-pin-nc:`nc` on-shield LCD i/f data :rpi-pico-sys:`LCD_DIN`
+       - | :rpi-pico-pin:`16` PIO/PWM : :rpi-pico-pio:`GP12` (PWM12)
+         | :rpi-pico-pin-nc:`nc` on-shield LCD reset :rpi-pico-sys:`LCD_RST`
+       - | :rpi-pico-pin:`17` PIO/PWM : :rpi-pico-pio:`GP13` :rpi-pico-pwm:`PWM13`
+         | :rpi-pico-pin-nc:`nc` on-shield LCD backlight :rpi-pico-sys:`LCD_BL`
+       - | :rpi-pico-pin:`18` :rpi-pico-gnd:`GND`
+       - | :rpi-pico-pin:`19` PIO/PWM : :rpi-pico-pio:`GP14` (PWM14)
+         | :rpi-pico-pin-nc:`nc` on-shield LDO power enable
+         | :rpi-pico-pin-nc:`nc` (optional) :rpi-pico-sys:`VCC_EN`
+       - | :rpi-pico-pin:`20` PIO/PWM : GP15 (PWM15)
+       - | :rpi-pico-pin:`21` PIO/PWM : GP16 (PWM0)
+       - | :rpi-pico-pin:`22` PIO/PWM : GP17 (PWM1)
+       - | :rpi-pico-pin:`23` :rpi-pico-gnd:`GND`
+       - | :rpi-pico-pin:`24` PIO/PWM : GP18 (PWM2)
+       - | :rpi-pico-pin:`25` PIO/PWM : GP19 (PWM3)
+       - | :rpi-pico-pin:`26` PIO/PWM : GP20 (PWM4)
+       - | :rpi-pico-pin:`27` PIO/PWM : GP21 (PWM5)
+       - | :rpi-pico-pin:`28` :rpi-pico-gnd:`GND`
+       - | :rpi-pico-pin:`29` PIO/PWM : GP22 (PWM6)
+       - | :rpi-pico-pin:`30` :rpi-pico-sys:`RUN` (RESET)
+       - | :rpi-pico-pin-nc:`nc` PIO/PWM : GP23 (PWM7)
+       - | :rpi-pico-pin-nc:`nc` PIO/PWM : GP24 (PWM8)
+       - | :rpi-pico-pin-nc:`nc` PIO/PWM : GP25 (PWM9)
+       - | :rpi-pico-pin:`31` ADC_CH0 : GP26 (PWM10)
+       - | :rpi-pico-pin:`32` ADC_CH1 : GP27 (PWM11)
+       - | :rpi-pico-pin:`33` :rpi-pico-gnd:`GND`
+       - | :rpi-pico-pin:`34` ADC_CH2 : GP28 (PWM12)
+       - | :rpi-pico-pin-nc:`nc` ADC_CH3 : GP29 (PWM13)
+       - | :rpi-pico-pin:`35` ADC_VREF
+       - | :rpi-pico-pin:`36` :rpi-pico-sys:`3V3(OUT)`
+       - | :rpi-pico-pin:`37` :rpi-pico-sys:`3V3_EN`
+         | :rpi-pico-pin-nc:`nc` on-shield LDO power enable
+         | :rpi-pico-pin-nc:`nc` (optional) :rpi-pico-sys:`VCC_EN`
+       - | :rpi-pico-pin:`38` :rpi-pico-gnd:`GND`
+       - | :rpi-pico-pin:`39` :rpi-pico-vdd:`VSYS`
+         | :rpi-pico-pin-nc:`nc` on-shield LDO power enable
+         | :rpi-pico-pin-nc:`nc` (default) :rpi-pico-sys:`VCC_EN`
+       - | :rpi-pico-pin:`40` :rpi-pico-sys:`VBUS`
+
+       .. rubric:: Devicetree compatible
+
+       - :dtcompatible:`raspberrypi,pico-header-r3`
+       - :dtcompatible:`gpio-keys`
```

Comment:
**praise (blocking):** remove link to `gpio-keys`, this shield has no input elements

Reply:
done